frontend/src/features/mapSideBar/tabs/SectionField.tsx

```diff
@@ -1,9 +1,11 @@
 import clsx from 'classnames';
+import TooltipIcon from 'components/common/TooltipIcon';
 import { Col, Row } from 'react-bootstrap';
 import styled from 'styled-components';
 
 interface ISectionFieldProps {
   label: string;
+  helpText?: string;
```

Comment:
Support showing a tooltip with "help text" next to the label on a section like shown below:

![image](https://user-images.githubusercontent.com/24322224/191641286-dd7c3d55-0a2a-46bf-85fc-426994e3d32c.png)